.vsts-dotnet.yml

```diff
@@ -1,5 +1,6 @@
 trigger:
 - master
+- vs16.0
```

Comment:
Should we just do `vs*`? "All release branches"?

Reply:
Does that work?